resource_customizations/serving.kserve.io/InferenceService/health_test.yaml

```diff
@@ -0,0 +1,13 @@
+tests:
+#- healthStatus:
+#    status: Progressing
+#    message: "PredictorConfigurationReady is Unknown\nPredictorReady is Unknown, since RevisionMissing. Configuration \"hello-world-predictor-default\" is waiting for a Revision to become ready.\nPredictorRouteReady is Unknown, since RevisionMissing. Configuration \"hello-world-predictor-default\" is waiting for a Revision to become ready.\nReady is Unknown, since RevisionMissing. Configuration \"hello-world-predictor-default\" is waiting for a Revision to become ready.\n"
+#  inputPath: testdata/progressing.yaml
+#- healthStatus:
+#    status: Degraded
+#    message: "IngressReady is False, since Predictor ingress not created.\nPredictorConfigurationReady is False, since RevisionFailed. Revision \"helloworld-00002\" failed with message: Container failed with: container exited with no error.\nPredictorReady is False, since RevisionFailed. Revision \"helloworld-00002\" failed with message: Container failed with: container exited with no error.\nReady is False, since Predictor ingress not created.\n"
+#  inputPath: testdata/degraded.yaml
```

Comment:
Wanna uncomment these and see if we get past tests?

Reply:
Yes. Just testing one by one. Once this healthy one passes I will uncomment others too.